src/openrct2/windows/guest_list.c

```diff
@@ -141,7 +141,7 @@ static sint32 _window_guest_list_highlighted_index; // 0x00F1EE10
 static sint32 _window_guest_list_selected_tab;      // 0x00F1EE12
 static sint32 _window_guest_list_selected_filter;   // 0x00F1EE06
 static sint32 _window_guest_list_selected_page;     // 0x00F1EE07
-static uint32 _window_guest_list_selected_view;  // 0x00F1EE13
+static uint32 _window_guest_list_selected_view;		// 0x00F1EE13
```

Comment:
GitHub / browser? doesn't display tabs correctly, it assumes all tabs are the same width - but really they should just move to the next tab stop (column that is multiple of tab size).